components/brave_wallet/common/common_utils.cc

```diff
@@ -347,4 +348,14 @@ std::string GetNetworkForZCashKeyring(const mojom::KeyringId& keyring_id) {
   NOTREACHED_NORETURN();
 }
 
+bool IsHTTPSOrLocalhostURLStr(const std::string& url_string) {
+  return IsHTTPSOrLocalhostURL(GURL(url_string));
+}
+
+bool IsHTTPSOrLocalhostURL(const GURL& url) {
```

Comment:
maybe this should be a more generic util, see https://github.com/brave/brave-core/pull/23916/files#r1643502696

Reply:
ok working on it

Reply:
Fixed in https://github.com/brave/brave-core/pull/24249/commits/180ace2b047e8971ecf97e7a5459d08685ba7f51